app/assets/javascripts/application.js

```diff
@@ -15,3 +15,9 @@
 //= require turbolinks
 //= require cartify/application
 //= require_tree .
+function openDiv(categories) {
```

Comment:
I can't see that you make use of the argument that was passed in... Am I missing something?